jupyterlab/labextensions.py

```diff
@@ -190,9 +190,12 @@ def run_task(self):
 class BuildLabExtensionApp(BaseExtensionApp):
     description = "Build labextension"
 
+    static_path = Unicode('', config=True,
```

Comment:
Do we want to default to `None`, since we are checking for `None` in `dynamic_labextensions.py`?

Reply:
(I think it works now - I think it just passes the empty string all the way through to the check against the empty string - it's more a question of style)